### ISSUE TITLE
Kebob dropdown inappropriate position on UI 

### ISSUE THREAD

**Issue by naina-verma** [almighty/almighty-ui#495]
Kebob dropdown inappropriate posiotion on UI 
- While clicking on WI from bottom , the kebob menu should be shown up on top side ,instead of showing down side(screenshot attached)
![screenshot from 2016-12-12 11-41-39](https://cloud.githubusercontent.com/assets/19172870/21089784/f1e15c12-c060-11e6-8d6f-90044be11404.png)

**Comment by naina-verma** [comment]
@fabric8io/uxd  please provide input on this issue , this is pending from long time !

**Comment by catrobson** [comment]
Correct, the direction of the pop-up from the kebab should switch based on whether there is enough vertical space to display it.  If there is enough vertical space below the kebab to display the full dropdown, it should drop down below the kebab.  If there is not enough space, it would drop "up" above the kebab.  This capability can be seen in the Angular 1.x implementation of the List View here: http://www.patternfly.org/pattern-library/content-views/list-view/#/api
